src/verify.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const childProcess = require('child_process');
+const debug = require('debug');
+const fs = require('fs');
+
+const log = debug('verify');
+const error = debug('verify:ERROR');
+// For some reason RED is 1.
+error.color = 1;
+
+function isBaselineAllPass(baselineFile) {
+  const lines = fs.readFileSync(baselineFile, 'utf-8').split(/\r?\n/g);
+  let allPass = true;
+  lines.forEach((line) => {
+    if (line.startsWith('FAIL') || line.startsWith('WARN')) {
+      allPass = false;
+    }
+  });
+  return allPass;
+}
+
+// transformedFile is a path to an html test
+// Assumes cwd is chromium/src/third_party/blink/web_tests/
+// Assumes content_shell is built and up to date.
+// Returns true if the transformedFile passses all tests, false otherwise.
+function verifyTransformation(transformedFile, targetBuild) {
+  if (process.cwd().split('/').pop() != 'web_tests') {
```

Comment:
wonder if this could just be `if (process.cwd().endswith(third_party/blink/web_tests)`